seml/manage.py

```diff
@@ -204,8 +216,21 @@ def delete_experiments(db_collection_name, sacred_id, filter_states, batch_id, f
                 if not yes and input('Are you sure? (y/n)').lower() != 'y':
                     exit()
             batch_ids_in_del = set([exp['batch_id']])
+
+            # Collect sources uploaded by sacred.
+            exp = collection.find_one({'_id': sacred_id}, {'experiment.sources': 1, 'artifacts': 1})
+            if 'experiment' in exp:
+                if 'sources' in exp['experiment']:
+                    exp_sources = exp['experiment']['sources']
+                    experiment_files_to_delete.extend([x[1] for x in exp_sources])
+                if 'artifacts' in exp:
+                    experiment_files_to_delete.extend([x['file_id'] for x in exp['artifacts']])
```

Comment:
This code is a duplicate from above. Refactor into separate function?

Reply:
Good idea. Done in the latest commit.